components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.v1/src/main/java/org/wso2/carbon/identity/api/server/organization/management/v1/service/OrganizationManagementService.java

```diff
@@ -662,4 +946,9 @@ private OrgApplicationManager getOrgApplicationManager() {
 
         return OrganizationManagementServiceHolder.getInstance().getOrgApplicationManager();
     }
+
+    private OrganizationDiscoveryManager getOrganizationDiscoveryManager() {
+
+        return OrganizationManagementServiceHolder.getInstance().getOrganizationDiscoveryManager();
```

Comment:
move this to a class variable.

Reply:
As discussed offline, will be keeping this as it is